acceptance/suites/tests/00_smoke/puppetdb_integration.rb

```diff
@@ -10,7 +10,14 @@
 ## PuppetDB successfully received the agent's report sent from Puppet Server.
 ## We can just run the agent that's on the master for this.
 
-confine :to, {:platform => puppetdb_supported_platforms}, master
+# This confine was originally
+# confine :to, {:platform => puppetdb_supported_platforms}, master
+# With , master present, execution unexpectedly enters the body of the
+# test when we expect the test to be skipped.  BKR-650 maybe relavent.
+# Be aware that future modifications to this test might, especially
+# if there are test operations added that run against agents instead of just
+# masters.
+confine :to, {:platform => puppetdb_supported_platforms}
```

Comment:
> I'm thinking you put ", master" on there intentionally

I did, because the code in here will install PuppetDB on the master, and if the master isn't a supported platform for PuppetDB, then we can't run this test.  It makes no sense without being tied to the master, but we don't need to necessarily use `confine` to tie it.

I thought I saw @erikPrime has refactored the conditional to not use `confine`, but instead just a `master.platform.include?(puppetdb_supported_platforms)` or something like that?  I think that would work here.

The buggy behavior is within `confine`, so we just want to avoid that method, but we definitely want to tie this to the master's platform somehow.